expan/core/early_stopping.py

```diff
@@ -105,97 +107,63 @@ def group_sequential(x,
     else:
         stop = False
 
+    #TODO: why return frequentist confidence interval in bayesian method
```

Comment:
regarding this comment: actually this is frequentist early stopping, not bayesian, so I will remove it.

Reply:
Oh I see. Sorry lol.